docs/UserGuide.md

```diff
@@ -343,8 +347,8 @@ Examples:
 - Sorting by name using `sort -s so/a sb/n` followed by `mark -s 1 inactive`
 
 Expected outputs:
-- Supplier at index 3 of the displayed list has status marked as `active`, assuming there is at least three suppliers displayed and the third supplier is not currently `INACTIVE`. Otherwise, an error message will be shown.
-- Supplier at index 1 of the displayed list has status marked as `inactive`, assuming there is at least one supplier displayed and the first supplier is not currently `INACTIVE`. Otherwise, an error message wil be shown.
+- Supplier at index 3 of the displayed list has status marked as `active`, assuming there is at least three suppliers displayed and the third supplier is not currently `inactive`. Otherwise, an error message will be shown.
```

Comment:
I thought it should be the supplier that is active cannot be marked as  active again